### ISSUE TITLE
Relevé sur les AP

### ISSUE THREAD

**Issue by jpm-cbna** [PnX-SI/gn_module_flore_prioritaire#43]
Lorsque l'on saisit une ZP/AP dans le module Bilan Stationnel, on n'a pas la possibilité de saisir les autres espèces dans le cas où l'on a réalisé un relevé. Cette fonctionnalité n'était pas non plus présente dans l'ancien module.
Actuellement, on doit donc saisir ce relevé dans notre outil de gestion d’occurrence de taxons. Dans ce cas, on intègre également l'espèce qui a fait l'objet du BS. On génère donc un doublon d'observation.
Comment cela est géré ? Est-il envisagé, pour la suite, de donner cette possibilité de saisir un relevé ? Cela éviterait la double saisie et le problème de doublons.

Remarque de @okristo.

**Comment by camillemonchicourt** [comment]
Le protocole prévoit que l'on ne prospecte qu'une espèce, donc faire un relevé d'observation d'autres taxons le semble hors périmètre du protocole et donc à faire ailleurs dans Occtax ou autre, indépendamment.

**Comment by jpm-cbna** [comment]
> à faire ailleurs dans OccTax ou autre, indépendamment.

Oui, c'est ce que j'imaginé. Mais du coup, il n'est pas possible d'indiquer que l'observation du taxon réalisé dans ce module "Flore Prioritaire" appartient au même relevé que celui renseigné dans OccTax par exemple.

L'observation provenant du module "Flore Prioritaire" qui apparaîtra dans la Synthèse sera considéré comme étant dans un relevé distinct avec une valeur dans le champ `unique_id_sinp_grp` différente de celle des observations saisies dans le module OccTax.
C'est plutôt ça le problème à considérer ici. Est il apparut dans d'autres modules de GeoNature ?

**Comment by camillemonchicourt** [comment]
Je pense au contraire que c'est bien et souhaitable.

Le taxon observé dans le cas de la prospection du protocole Flore prioritaire est à part, avec la logique du protocole.
Les autres taxons sont distincts, dans un relevé classique Occtax.

**Comment by camillemonchicourt** [comment]
Autre élément qui pousse à les séparer. 
Ce sont des contextes et protocoles différents, donc des JDD différents.

Hors on ne peut pas avoir 2 observations d'un même relevé dans des JDD différents.
